platform/core/src/types/HangingProtocol.ts

```diff
@@ -293,6 +293,16 @@ export type Protocol = {
    */
   numberOfPriorsReferenced?: number;
   syncDataForViewports?: boolean;
+  /* Set of minimal conditions necessary to run the hanging protocol
```

Comment:
Please add a newline before this so it gets read correctly by the comment docs extractor.

Reply:
Updated, thanks.